internal/encoding/typeof/typeof.go

```diff
@@ -81,6 +84,13 @@ func FromOrc(desc *orc.TypeDescription) (Type, bool) {
 	return t, ok
 }
 
+// FromParquet maps the parquet type description to our type.
+func FromParquet(desc *parquet.Type) (Type, bool) {
+	kind := desc.String()
+	t, ok := supported[kind]
```

Comment:
I wonder if we need to have 2 maps, one for `orc` and another for `parquet`. Otherwise it may be a bit confusing.

Reply:
Is it really needed, as long as the map is a superset of the types we support?

Reply:
True, but let's add a comment on each type to see what is supported by which decoder. Something like:
```
	"INT32":      Int32, // Parquet
	"INT64":      Int64, // Parquet
        "DOUBLE": Float64, // ORC
        "STRING": String, // ORC, Parquet
```